### PR TITLE
curve_tool: fix log

### PR DESCRIPTION
**Pull request by SeanHai**
### What problem does this PR solve?

Issue Number: close #xxx 

Problem Summary:

### What is changed and how it works?

What's Changed:

How it Works:

Side effects(Breaking backward compatibility? Performance regression?): 

### Check List

- [ ] Relevant documentation/comments is changed or added
- [ ] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

File: tools/curvefsTool.cpp [hunk 1]
```diff
@@ -233,16 +124,28 @@ int CurvefsTools::TryAnotherMdsAddress() {
     return ret;
 }
 
+int CurvefsTools::DealRedirectMdsRet(int ret, std::string operation) {
+    if (kRetCodeRedirectMds == ret) {
+        LOG(ERROR) << operation << " fail on mds: "
```
[the remark this answers is not in the document]
Comment (SeanHai) [reply tools/curvefsTool.cpp]: > Recommended to use 'warning'

done

**Comment by SeanHai** [comment]
recheck

File: tools/curvefsTool.cpp [hunk 2]
```diff
@@ -233,16 +124,28 @@ int CurvefsTools::TryAnotherMdsAddress() {
...
-        return ret;
+        return DealFailedRet(ret, "read cluster map");
     }
     ret = InitLogicalPoolData();
     if (ret < 0) {
-        LOG(ERROR) << "init logical pool data fail";
-        return ret;
+        return DealFailedRet(ret, "init logical pool data");
+    }
+    ret = ScanLogicalPool();
```
Comment (xu-chaojie) [inline tools/curvefsTool.cpp:146]: Did it miss 'ScanLogicalPool' before?

Comment (SeanHai) [reply tools/curvefsTool.cpp:146]: > Did it miss 'ScanLogicalPool' before?

yes

**Comment by SeanHai** [comment]
recheck

**Review by xu-chaojie — APPROVED** [review]

**Review by ilixiaocui — APPROVED** [review]

**Pull request merged**